app/views/booking_requests/step_two.html.erb

```diff
@@ -5,13 +5,14 @@
 
     <% if @booking_request.errors.any? %>
       <div class="error-summary t-errors" role="group" aria-labelledby="error-summary-heading-example-1" tabindex="-1">
-        <h1 class="heading-medium error-summary-heading" id="error-summary-heading-example-1">
-          There was a problem with your details
-        </h1>
+        <h2 class="heading-medium error-summary-heading" id="error-summary-heading-example-1">
+          Unable to submit the form
+        </h2>
+        <p>Check the following:</p>
 
         <ul class="error-summary-list">
-          <% @booking_request.errors.full_messages.each do |msg| %>
-            <li><%= msg %></li>
+          <% @booking_request.errors.each do |key, error_msg| %>
```

Comment:
Can we change the second yielded variable to `message`? It has enough context so doesn't need the `error` prefix and abbreviating variables is a no-no despite it being `msg` before.